### ISSUE TITLE
`HOMETEL` path is hardcoded in `test_examples.py`

### ISSUE THREAD

**Issue by pmav99** [tomsail/telemac-conda#15]
`/home/tomsail` doesn't exist on most machines. Furthermore not everyone puts their code under `$HOME/apps` :)

https://github.com/tomsail/telemac-conda/blob/6854428a1ddb627f0a30895cbce54de243f503bd/tests/test_examples.py#L5

A more generic/portable way to get the path to the root of the repository is to use `__file__`: https://stackoverflow.com/questions/9271464/what-does-the-file-variable-mean-do

For example:

``` python
import os.path

current_module = os.path.abspath(__file__)
tests_dir = os.path.dirname(current_module)
root_dir = os.path.dirname(tests_dir)

print(current_module)
print(tests_dir)
print(root_dir)
```

For the record, you can also use the [`pathlib` module](https://realpython.com/python-pathlib/) which is the newer way for doing path manipulations in python:
``` python
import pathlib

current_module = pathlib.Path(__file__).resolve()
tests_dir = current_module.parent
root_dir = tests_dir.parent 

# or more succinctly:
# root_dir = pathlib.Path(__file__).resolve().parent.parent

print(current_module)
print(tests_dir)
print(root_dir)
```
Nevertheless, do notice that pathlib objects are **not** strings and will require further changes/updates in the rest of the script. If you are not interested in reading about `pathlib`, just use `os.path`.

**Comment by tomsail** [comment]
Thanks Panos :) 
I cannot close this issue with a PR, but see my fix here: 
https://github.com/tomsail/telemac-conda/blob/4535218fed44d46af39a1401536685307a891e56/tests/test_examples.py#L5C1-L5C31

**Comment by pmav99** [comment]
If `HOMETEL` is defined upon the environment's activation, then yes it is more than good enough.

BTW, what is the problem with closing the issue with a PR or a specific commit? You just need to add a certain string in the commit message. For example, any of the following would close the current issue:
```
fix #15
fixes #15
close #15
closes #15
```
For more info: https://github.blog/2013-01-22-closing-issues-via-commit-messages/

**Comment by tomsail** [comment]
fix #14
